app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+
+    if len(command_parts) != 3:
+        raise ValueError("Invalid command format")
+
+    _, source_file, destination_file = command_parts
+
+    if source_file == destination_file:
+        return
+
+    with open(source_file, "r") as file_in, \
+            open(destination_file, "w") as file_out:
```

Comment:
better to use `()` then `\`

Reply:
where exactly? I'd try but in every cases was a mistake

Reply:
with (
        open(file_to_copy, "r") as file_in,
        open(new_file, "w") as file_out
    ):